CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing Overview
+
+You are always welcome to contribute and Thank you for your help in improving WasmEdge :)
+
+This project values community collaboration and welcomes contributions from all interested parties. If you are new to the project, please review the newcomers welcome guide to learn more about how and where to contribute. Additionally, all contributors are expected to follow the Code of Conduct.
+
+If you are ready to contribute, please search for open issues labeled "help-wanted" and choose one to work on. Feel free to join our Slack channel to participate in discussions or create a new issue if you have a suggestion or request. When submitting a pull request, please reference the corresponding open issue if one exists. To automatically close related issues in GitHub, include keywords in your pull request descriptions and commit messages.
+
+**Contents**
+
+- <a name="contributing-flow">Contribution Flow</a>
+  - <a href="#commit-signoff">Developer Certificate of Origin</a>
+- WasmEdge Contribution Flow
+  - <a href="#contributing-docs">WasmEdge Docs</a> 
+  - <a href="https://wasmedge.org/book/en/contribute/contribute.html">WasmEdge Runtime</a>
+  - <a href="https://github.com/WasmEdge/WasmEdge/blob/master/SECURITY.md">Reporting a vulnerability</a>
+
+# <a name="contributing-flow">Contribution Flow</a>
+
+To contribute to WasmEdge, have a look at [here](https://wasmedge.org/docs/contribute/overview)
```

Comment:
Contributing to WasmEdge is different from contributing to WasmEdge docs. Let's specify the goal of this contributing guide.

Reply:
Oops! My bad, thanks for pointing it out :)